content/english/facts/soy-is-bad-for-you.md

```diff
@@ -0,0 +1,223 @@
+
+*Note:* This article suggests soy protein isolate has a stimulator effect on the growth of breast tissue, not _cancer_ tissue specifically
+
+#### Stimulatory Influence of Soy Protein Isolate on Breast Secretion in Pre- and Postmenopausal Women - 1996
+
+> Instead, this pilot study indicates that prolonged consumption of soy protein isolate has a stimulatory effect on the premenopausal female breast, characterized by increased secretion of breast fluid, the appearance of hyperplastic epithelial cells, and elevated levels of plasma estradiol. These findings are suggestive of an estrogenic stimulus from the isoflavones genistein and daidzein contained in soy protein isolate.  
+> - [Stimulatory Influence of Soy Protein Isolate on Breast Secretion in Pre- and Postmenopausal Women - 1996](http://web.archive.org/web/20191009183549/https://cebp.aacrjournals.org/content/cebp/5/10/785.full.pdf#zoom=75)
+
+*Note:* This article suggests soy protein isolate has a stimulator effect on the growth of breast tissue, not _cancer_ tissue specifically
+
+### Male Fertility
+
+#### Hypogonadism and erectile dysfunction associated with soy product consumption - 2011
+> We present the case of a 19-y-old type 1 diabetic but otherwise healthy man with sudden onset of loss of libido and erectile dysfunction after the ingestion of large quantities of soy-based products in a vegan-style diet. Blood levels of free and total testosterone and dehydroepiandrosterone (DHEA) were taken at the initial presentation for examination and continuously monitored up to 2 y after discontinuation of the vegan diet. Blood concentrations of free and total testosterone were initially decreased, whereas DHEA was increased. These parameters normalized within 1 y after cessation of the vegan diet.  
+> - [Hypogonadism and erectile dysfunction associated with soy product consumption - 2011](https://web.archive.org/web/20110305114701/https://www.ncbi.nlm.nih.gov/pubmed/21353476) 
+
+**Note:** This is a study of a single person which has not been replicated in any other study we have seen.
```

Comment:
Yeah. China :)    This article for a real debunking somewhere.

This is also interesting : https://www.mdpi.com/2072-6643/8/12/754

Reply:
Yeah that's a Mark Messina special though ;)